Documentation/ExtensionArchitecture/ComposerJson/Index.rst

```diff
@@ -86,6 +90,10 @@ Subsequently:
    You can add other system extensions and third party extensions,
    if your extension depends on them.
 
+``replace``
+  The replace part can be used to tell the composer that any other found code for your extension will be 
```

Comment:
This sounds odd. I'd recommend to rephrase this based on [the official explanation](https://getcomposer.org/doc/04-schema.md#replace).